.gitignore

```diff
@@ -25,15 +25,33 @@ tests/ccl_test_power
 tests/ccl_test_utils
 tests/check_ccl
 
+# Ignores compiled examples
+examples/CMakeFiles
+examples/cmake_install.cmake
+examples/ccl_sample_baryons
+examples/ccl_sample_neutrinos
+examples/ccl_sample_photoz
+examples/ccl_sample_pkemu
+examples/ccl_sample_power
+examples/ccl_sample_run
+
 # Ignore Python pyc, autogenerated swig files, and build dirs
 *.pyc
 build/*
+dist/*
 tests/.ipynb_checkpoints/
 *ipynb*
 *log
 *dat
 *bak
 *dylib
+pyccl/ccllib.py
+pyccl/ccl_params.ini
```

Comment:
Not sure why this file is in .gitignore -- it's a config file.

Reply:
Because these are regenerated automatically on build, on the local machine, they should not be stored on git

Reply:
(for the .ini file, it is copied over by from the include directory on build)